tests/bin/create-test-db.php

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env php
+<?php
```

Comment:
Don't we need licence header for this file ?

Reply:
Fixed